src/player.h

```diff
@@ -1515,6 +1515,7 @@ class player : public Character
         // Auto move methods
         void set_destination( const std::vector<tripoint> &route,
                               const player_activity &destination_activity = player_activity() );
+        void destination_activity_value( const int value );
```

Comment:
Where is this function implemented?

Reply:
Nowhere, thats a leftover, ill remove it.